rateapp/postcss.config.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
+    plugins: [
+        require('tailwindcss'),

```

Comment:
Ken ik niet. Is dit een aanrader? Ik leer graag iets bij (angel)

Reply:
Ik vind Tailwind vooral fijn voor het snel opbouwen van kleinere projectjes. Het is heel lightweight met alleen maar utility classes en een makkelijke config. Met frameworks als Bootstrap komt al heel snel heel veel troep mee die je nooit gaat gebruiken in de kleinere projectjes.

Ik heb nu 2 projecten met TailwindCSS opgezet en het bevalt me goed. Het duurt wel weer even voor je aan alle classes gewend bent, maar ze zijn redelijk intuitief als je een beetje CSS kan. Maar zoals met alles, altijd even kijken of het aansluit bij wat je wil gaan bouwen :-)